services/email/index.js

```diff
@@ -5,7 +5,7 @@ var nodemailer = require('nodemailer');
 var config = require('../../config');
 var i18nFuture = require('hof').i18n;
 var Hogan = require('hogan.js');
-var i18nLookup = require('hof').i18nLookup;
+var i18nLookup = require('i18n-lookup');
```

Comment:
Hmmm, I'm confused. So i18n-lookup is a utility for "looking-up" stuff from a bunch of translations and is not part of HOF, i18n-future is a "translations library" (whatever the fuck that means), and we use both because they depend on each other for some things. Seems to me like these two modules should be wrapped up together and exported as part of hof